Makefile

```diff
@@ -0,0 +1,4 @@
+all: .gitmodules
+
+.gitmodules:
+	git-setup.sh api website console
```

Comment:
If there's no `.gitmodules` file it will run the setup on `make`. This prevents running this command more than necessary.